src/OpenRiaServices.Tools/Test/CreateRiaClientFilesTaskTests.cs

```diff
@@ -283,6 +284,51 @@ public void CreateRiaClientFiles_Missing_Pdb_Warns()
             }
         }
 
+#if !NETFRAMEWORK
+        [TestMethod]
+        public void CreateRiaClientFiles_For_Net_60()
+        {
+            var rootPath = new DirectoryInfo(Directory.GetCurrentDirectory()).Parent.Parent.Parent.FullName;
+            var serverProjectPath = Path.Combine(rootPath,  "ServerClassLib\\ServerClassLib.csproj");
+            var clientProjectPath = Path.Combine(rootPath, "ClientClassLib\\ClientClassLib.csproj");
+            
+            var asmPath = "bin\\Debug\\net6.0";
+            string[] serverAsm = GetAssemblies(serverProjectPath, asmPath);
+            string[] clientAsm = GetAssemblies(clientProjectPath, asmPath).Where(c => !c.Contains("Client")).ToArray(); // Exclude client asm since they should not being built yet
+            var code = CodeGenHelper.CreateOpenRiaClientFilesTaskInstance(serverProjectPath, clientProjectPath, false, serverAsm, clientAsm);
+
+            var task = code.Execute();
+            Assert.IsTrue(task);
+            Assert.IsTrue(Directory.Exists(code.GeneratedCodePath));
+        }
+#endif
+
+
+        private static string[] GetAssemblies(string projectPath, string path)
```

Comment:
Remove 
* this helper 
* * and the possiblily to add additional files names to `CreateOpenRiaClientFilesTaskInstance`
* and CreateRIA60_TS